rootfs/api/models/__init__.py

```diff
@@ -142,6 +143,12 @@ def _log_config_updated(**kwargs):
     config.app.log("config {} updated".format(config))
 
 
+def _log_app_settings_updated(**kwargs):
+    appSettings = kwargs['instance']
+    # log only to the controller; this event will be logged in the release summary
+    appSettings.app.log("application settings {} updated".format(appSettings))
```

Comment:
this is logged also to the application, which means the user will see this log as well as the summary. Do you mean to call `logger.info()` instead?